tensorboard/webapp/routes/core_deeplink_provider.ts

```diff
@@ -112,15 +122,22 @@ export class CoreDeepLinkProvider extends DeepLinkProvider {
     queryParams: SerializableQueryParams
   ): DeserializedState {
     let pinnedCards = null;
+    let smoothing = null;
     for (const {key, value} of queryParams) {
-      if (key === 'pinnedCards') {
-        pinnedCards = extractPinnedCardsFromURLText(value);
-        break;
+      switch (key) {
+        case 'pinnedCards':
```

Comment:
Unrelated to PR, but since we are here, could we also extract `PINNED_CARDS_KEY`?